src/Loader.php

```diff
@@ -154,11 +154,28 @@ public function load_commands() {
 			return;
 		}
 
-		$command = static function ( $args, $assoc_args ) {
+		$activator_command = static function ( $args, $assoc_args ) {
 			Activator::run();
 		};
+		WP_CLI::add_command( 'p4-run-activator', $activator_command );
+
+		/**
+		 * Put the CF API key into the options table, where the CF plugin uses it from.
+		 */
+		$put_cf_key_in_db = static function ( $args ) {
+			$hostname = $args[0];
+			if ( empty( $hostname ) ) {
+				WP_CLI::error( 'Please specify the hostname.' );
+			}
+
```

Comment:
We could add an error if `CLOUDFLARE_API_KEY` is undefined (assuming activating the following options would cause some problems with a key missing)

Reply:
@lithrel Added the check :+1: A missing/wrong/expired key should only result in the purge request failing.

Reply:
https://github.com/cloudflare/Cloudflare-WordPress/blob/91ba8ddd4c9478d1615e5067d0759e4af6e8849e/src/WordPress/Hooks.php#L168-L174